.github/workflows/publish.yml

```diff
@@ -38,12 +27,15 @@ jobs:
       - name: Extract tag from ref
         if: startsWith(github.ref, 'refs/tags/')
         id: get_tag
-        run: echo ::set-output name=VERSION::$(echo $GITHUB_REF | sed "s/refs\/tags\///")
+        run: echo "VERSION=$(echo $GITHUB_REF | sed "s/refs\/tags\///")" >> $GITHUB_OUTPUT
```

Comment:
I think you can make this line a bit simpler:

```
echo VERSION=${GITHUB_REF#refs/tags/} >> $GITHUB_OUTPUT
```

Reply:
Or even simpler, remove this step entirely, and then use the `github.ref_name` to tell us the tag name.  https://docs.github.com/en/actions/learn-github-actions/contexts